db/migrate/20190305150816_drop_elements_from_vm_attrs_hash.rb

```diff
@@ -0,0 +1,68 @@
+class DropElementsFromVmAttrsHash < ActiveRecord::Migration[5.1]
+  def up
+    say "Formatting nics attributes, total #{FakeNic.unscoped.count}"
+    FakeNic.unscoped.each do |compute_attribute|
+      drop_elements(compute_attribute.attrs)
```

Comment:
this and below should be `compute_attribute.attrs = drop_elements(compute_attribute.attrs)` or the top level won't be fixed.